### PR TITLE
feat: update post navigation per design changes

### PR DESCRIPTION
**Pull request by capJavert**
### Describe what this PR does
Before:
image
After:
image

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1447 #done

### CODE REVIEW

File: packages/shared/src/components/post/PostNavigation.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ function PostNavigation({
       {onPreviousPost && (
         <SimpleTooltip content="Previous">
           <Button
-            className="-rotate-90 btn-secondary"
+            className="border-none -rotate-90 btn-secondary"
```
Comment (rebelchris) [inline packages/shared/src/components/post/PostNavigation.tsx:53]:
```suggestion
            className="-rotate-90 btn-tertiary"
```

I think this should do looking at the design spec.

File: packages/shared/src/components/post/PostNavigation.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ function PostNavigation({
       {onNextPost && (
         <SimpleTooltip content="Next">
           <Button
-            className="rotate-90 btn-secondary"
+            className="border-none rotate-90 btn-secondary"
```
Comment (rebelchris) [inline packages/shared/src/components/post/PostNavigation.tsx:65]:
```suggestion
            className="rotate-90 btn-tertiary"
```

**Review by rebelchris — CHANGES_REQUESTED** [review]

File: packages/shared/src/components/post/PostNavigation.tsx (hunk 1 above)
Comment (capJavert) [reply packages/shared/src/components/post/PostNavigation.tsx:53]: Ah sorry about that. Updated. I would expect `Ter` to be up here also, but it is not. 🤦 

image

**Review by rebelchris — APPROVED** [review]
Wohoo, you can probably even go directly to main for this one actually doesn't seem breaking changes 👍

**Pull request merged**